index.html

```diff
@@ -57,31 +57,31 @@ <h1 id="primary">Hey There. <br />I'm Lee</h1>
       <ul id="socialmedia" class="flexs">
         <li>
           <a class="icons" href="https://github.com/leehaney254"
-            ><img src="./assets/github.png" alt="Github link"
+            ><img src="./assets/github.png" alt="Link to my github page"
           /></a>
         </li>
         <li>
           <a
             class="icons"
             href="https://www.linkedin.com/in/leehaney-george-0a4a51178/"
-            ><img src="./assets/Linkedin.png" alt="Linkedin icon"
+            ><img src="./assets/Linkedin.png" alt="Link to my linkedin page"
           /></a>
         </li>
         <li>
           <a
             class="icons"
             href="https://app.slack.com/client/T47CT8XPG/D0487PRDM7F/rimeto_profile/U03T2ES9771"
-            ><img src="./assets/slack.png" alt="Slack icon"
+            ><img src="./assets/slack.png" alt="Link to my slack page"
           /></a>
         </li>
         <li>
           <a class="icons" href="https://twitter.com/Lee06785586"
-            ><img src="./assets/twitter.png" alt="Twitter icon"
+            ><img src="./assets/twitter.png" alt="Linkto my twitter page"
           /></a>
         </li>
         <li>
           <a class="icons" href="georgeleehaney@gmail.com"
-            ><img src="./assets/gmail.png" alt="gmail icon"
+            ><img src="./assets/gmail.png" alt="Link to my gmail page"
```

Comment:
Modified the image alt text to be more descriptive and null for decoration. :100: